receiver/oracledbreceiver/obfuscate.go

```diff
@@ -0,0 +1,35 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// source: https://github.com/DataDog/datadog-agent/blob/main/pkg/collector/python/datadog_agent.go
```

Comment:
No need to bring `sync` usage from datadog examples. Can you please apply https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/41214 here as well?